tsconfig.json

```diff
@@ -8,6 +8,5 @@
     "resolveJsonModule": true,
     "jsx": "react-jsx",
     "allowSyntheticDefaultImports": true,
-    "sourceMap": true,
   }
```

Comment:
Removed it because it causes larger build size. But this can be useful in debugging in future. (Something I learned)